AQA/src/register.spec.ts

```diff
@@ -0,0 +1,136 @@
+describe("Registration", () => {
+  const url = "https://anatoly-karpovich.github.io/demo-login-form/";
+  
+
+  before(async () => {
+    await browser.maximizeWindow();
+  });
+  beforeEach(async () => {
+    await browser.url(url);
+  });
+  context("Positive scenarious", () => {
+    const validCredentials = {
```

Comment:
Only two cases for registration :(
You can create prepared object with valid and notValid arrays with credentials
Use `for...of` to iterate through test data and set them to inputs

Reply:
This was learned only on the next lection^)

Reply:
I was thinking from the point of view that we already know how to work with arrays and objects :(